osbs/utils.py

```diff
@@ -331,6 +331,44 @@ def utcnow():
 
 
 VALID_BUILD_CONFIG_NAME_CHARS = re.compile('[-a-z0-9]')
+VALID_LABEL_CHARS = re.compile('[-a-z0-9\/\.]')
```

Comment:
We probably don't want to include `/` because label keys can only include that once, and anything before the `/` has to be valid for a DNS subdomain.